src/Symfony/Component/Lock/QuorumInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock;
+
+/**
+ * QuorumInterface defines an interface to indicate when a quorum is met and can be met.
+ *
+ * @author Jérémy Derussé <jeremy@derusse.com>
+ */
+interface QuorumInterface
+{
+    /**
+     * Returns Whether or not the quorum is met.
+     *
+     * @param int $numberOfSuccess
+     * @param int $numberOfItems
+     *
+     * @return bool
+     */
+    public function isMet($numberOfSuccess, $numberOfItems);
+
+    /**
+     * Returns Whether or not the quorum *could* be met.
+     * This method does not means the quorum *would* be met for sur. But can be useful to early stop a process when you
+     * known there isn't any chance to met the quorum.
+     *
```

Comment:
I think there should be a blank line between the main and sub description

This method does not mean~~s~~ the quorum \*would\* be met for ~~sur. But~~ sure, but can be useful to ~~early~~ stop a process **early** when you known there ~~isn't any~~ **is no** chance to me**e**t the quorum.